physlr/physlr.py

```diff
@@ -2724,8 +2670,17 @@ def parse_arguments():
             help="ARCS scaffold pairing distance estimation file.")
         argparser.add_argument(
             "--dist-type", action="store", dest="dist_type", type=str, default="avg",
+            choices=["min", "avg", "max"],
             help="ARCS scaffold pairing distance type."
-                 "Acceptable values are: min, avg, max")
+                 "Accepted values are: min, avg, or max [avg].")
+        argparser.add_argument(
+            "--map-type", action="store", dest="map_type", type=str, default="all",
+            choices=["all", "split"],
```

Comment:
```suggestion
            choices=["unsplit", "split"],
```
Trying to think of alternatives to `all` -- because it's not like we are not using all the minimizers in split, just split vs unsplit